### DISCUSSION TITLE
Unnecessarily pestilence cutscene keeps playing

### DISCUSSION THREAD

**Discussion by CybernetHacker14** [Keriew/augustus#632] [Q&A]
Not sure whether this is an issue, but the Pestilence cutscene keeps playing from time to time on my fully built city. It also has a good network of clinics, barbers and bathhouses. The only thing not having a good coverage is the hospital.

Until version 3.0.1, I didn't encounter this issue. I am playing the 3.1.0 custom built version (i.e. cloned the source code and executing the build through Visual Studio). I didn't make any changes to the doctors' or the surgeon's code, hence not sure what is actually happening here. Also, the pestilence cutscene plays, but I don't see any fumigation action happening, and buildings being set on fire, which is usually the case.

Just wanted to know, whether this is deliberately built into any developmental versions, or is this actually a new feature in 3.1.0 that hospitals are also necessary in order to not have the pestilence situation happening?

**Comment by crudelios** [comment]
Yeah, you're probably running a that is a few months old and hospitals were required to avoid pestilence. Later builds changed this mechanic so hospitals are not as important.

**Reply by CybernetHacker14** [reply]
Gotcha. So which branch is then the latest one? Cause I had forked it off `master` and there is also a `release` branch.

**Reply by crudelios** [reply]
It's the `master` branch. What's your specific build version? It should don't annoy you that muah unless overall health is low enough.

**Reply by CybernetHacker14** [reply]
Specific build version is 3.1.0 only, atleast that's what is showing up in the menu.

**Reply by crudelios** [reply]
Ah then it's the latest release? You can try downloading the latest unstable build and see if it improves things. Just beware that saves from the latest unstable won't open on 3.1.0 (they will open on 3.2.0 when it's finally released though)

**Reply by CybernetHacker14** [reply]
Alright

**Reply by CybernetHacker14** [reply]
> It's the `master` branch. What's your specific build version? It should don't annoy you that muah unless overall health is low enough.

Doesn't annoy in the beginning, but in later stages, it starts showing up frequently, like within intervals of half a minute.

**Reply by Ouaz** [reply]
> or is this actually a new feature in 3.1.0 that hospitals are also necessary in order to not have the pestilence situation happening?

It was added temporarily a few unstable builds ago and has been removed since then. What happened I think:

You didn't have enough hospitals in your fully built city, therefore your hospital coverage was bad. This applied a malus to the sickness decrease (the malus was bigger as the city pop increased, that's why you didn't have problems in the beginning), and surely some of your trade buildings get plagued relatively often, and houses from time to time maybe. But as you said you had a good network of clinics (so doctors were likely very close), it's possible the fumigation ended before you checked the trouble location (moreover if you played with a high speed setting, as the fumigation lasts only a few days).

Whatever, the health calculation/sickness mechanic has been fully revamped for incoming 3.2 (for example, hospital are a bonus now to fight sickness, among other changes). This will be fully documented in the 3.2 manual.

Thanks for your feedback and, if you play with the latest unstable build, feel free to report some weird behavior regarding health/sickness, if you encounter some (it _should_ be ok now, though ^^).
